addons/payment/controllers/portal.py

```diff
@@ -318,6 +312,25 @@ def _create_transaction(
 
         return tx_sudo
 
+    @staticmethod
+    def _update_landing_routes(tx_sudo):
+        """ Recompute the access token stored in the landing routes of validation transactions.
```

Comment:
```suggestion
        """ Recompute the access token stored in the landing routes of validation transactions.
        
```
This helps IDEs show a short docstring, similar to commit title vs. message body.